core/domain/translation_domain_test.py

```diff
@@ -197,6 +197,16 @@ def test_unregistered_translatable_object_raises_exception(self) -> None:
             Exception, 'Must be implemented in subclasses.'):
             translatable_object.get_translatable_contents_collection()
 
+    def test_registering_duplicate_content_id_raises_exception(self) -> None:
```

Comment:
It wasn't clear to me why creating a "regular" DummyTranslatableObject in this way would lead to an error. I think you might need a more descriptive class name here that encapsulates what is explained in that class's docstring.

Reply:
Updated class name to `DummyTranslatableObjectWithDuplicateContentIdForParams`